website/docs/reference/programmatic-invocations.md

```diff
@@ -91,7 +91,7 @@ Register `callbacks` on dbt's `EventManager`, to access structured events and en
 
 ```python
 from dbt.cli.main import dbtRunner
-from dbt.events.base_types import EventMsg
+from dbt_common.events.base_types import EventMsg
```

Comment:
hello @pgoslatara : looks like we'll need to have two copies of this python example. one for v1.5 - v.17 and the other for v1.8 and later.  

can you update your PR so it looks like this following:  

`<VersionBlock firstVersion="1.8">`

Copy the entire python code block here except use your dbt_common change

`</VersionBlock>`

`<VersionBlock lastVersion="1.7">`

Copy the entire python code block here, exactly how it looks right now

`</VersionBlock>`

Reply:
@pgoslatara : here's a section that contains an example of using `VersionBlock` that's published https://github.com/dbt-labs/docs.getdbt.com/blob/ccaf6b9877e378cde258955e27366d2b181ce9fd/website/docs/docs/core/pip-install.md?plain=1#L40

Reply:
Thanks @nghi-ly for the helpful reference! Feedback implemented in e761a65.